### PR TITLE
Feature/responsive/add-media-queries

### PR DESCRIPTION
**Pull request by just-s-i-d**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1,92 +1,113 @@
...
+            </nav>
+        </header>
+        <section class="about-section">
+            <div class="about-container">
+                <h1>About ePrescripto
+                    <hr>
+                </h1>
 
-<h1>Our Vision <hr></h1>
+                Welcome to ePrescripto, your trusted partner in prescription management and healthcare optimization.
```
Comment (karangkm) [inline about.html:38]: Have practice to put content into some typographic tags like `<p />`, `<span />`, etc.

File: css/style.css [hunk 2]
```diff
@@ -337,46 +355,112 @@ button {
...
+.step-heading {
     color: var(--green);
     font-size: 22px;
-    margin: 20px 0 10px;
+    margin: 20px 0 10px 20px;
 }
 
-.how-to-use-section .step-des {
+.step-heading .step-des {
+    color: #252525;
```
Comment (karangkm) [inline css/style.css:405]: Try to use colors in variables.

File: css/style.css [hunk 3]
```diff
@@ -431,25 +533,277 @@ font-size: 48px;
     bottom: 0;
 }
 
+/* media queries starts here */
+@media (max-width:1350px) {
```
Comment (karangkm) [inline css/style.css:544]: Recommend to use breakpoints which are community accepted.

File: css/style.css [hunk 4]
```diff
@@ -431,25 +533,277 @@ font-size: 48px;
...
+        font-size: 22px;
+        margin: 20px 0 10px;
+    }
+
+    .how-to-use-section .step-des {
+        width: 70%;
+    }
+}
+
+@media (max-width:820px) {
```
Comment (karangkm) [inline css/style.css:668]: Avoid using `max-width` with media queries always try to implement using `min-width`

File: about.html (hunk 1 above)
Comment (just-s-i-d) [reply about.html:38]: ok

**Pull request merged**